iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/variant_internal.hpp

```diff
@@ -230,14 +230,14 @@ struct call_at_index<N, T>
         }
     }
 
-    // 'operator new()' needs non-const 'destination'
-    // NOLINTNEXTLINE (readability-non-const-parameter)
+    // NOLINTJUSTIFICATION 'operator new()' needs non-const 'destination'
```

Comment:
`NOLINTJUSTIFICATION` could be also added to lines 171, 187 and 203. But to be honest, I don't know if we usually insist on it.